lib/rack/lint.rb

```diff
@@ -304,14 +317,12 @@ def check_environment(env)
         ## is reserved for use with the Rack core distribution and other
         ## accepted specifications and must not be used otherwise.
         ##
-        %w[REQUEST_METHOD SERVER_NAME QUERY_STRING SERVER_PROTOCOL rack.errors].each do |header|
-          raise LintError, "env missing required key #{header}" unless env.include? header
-        end
```

Comment:
This should be part of the first commit, since you are replacing this with separate calls to `assert_required`.

Reply:
Good call, done.